### ISSUE TITLE
Unable to send mobile report

### ISSUE THREAD

**Issue by afalko** [Lab-Brat/gentoo_update#21]
I ran an update and then tried to send a mobile report after setting GU_TOKEN, but I see this: 
```
# gentoo-update --send-report=mobile
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.11/gentoo-update", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/gentoo_update.py", line 219, in main
    report = generate_last_report(log_dir).create_report()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/gentoo_update.py", line 182, in generate_last_report
    update_info = Parser(last_log).extract_info_for_report()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/parser.py", line 371, in extract_info_for_report
    log_info.pretend_emerge = self.parse_emerge_pretend_section(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/parser.py", line 130, in parse_emerge_pretend_section
    pretend_details = self.parse_pretend_details(section_content)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/gentoo_update/parser.py", line 195, in parse_pretend_details
    error_index = section_content.index(
                  ^^^^^^^^^^^^^^^^^^^^^^
ValueError: 'emerge pretend has failed, exiting' is not in list
```

**Comment by Lab-Brat** [comment]
Did you run security updates only (`gentoo-update`) or full updates (`gentoo-update -m full`) on the previous run?

It would be helpful to see the last update log, which is in `/var/log/portage/gentoo-update/` by default. Looks like there might be some parsing error. You can attach the log here or send it to me directly via labbrat_social@pm.me

**Comment by afalko** [comment]
I tried it after security update and also after doing nothing.

**Comment by Lab-Brat** [comment]
@afalko Fixed in #22 (0.1.14)

Please note that `gentoo-update --send-report mobile` still won't work with your previous log file.
But it should work after you re-run `gentoo-update`
